src/engine/client/cl_main.cpp

```diff
@@ -3467,7 +3467,7 @@ void CL_LocalServers_f()
 		// can nicely run multiple servers
 		for ( j = 0; j < NUM_SERVER_PORTS; j++ )
 		{
-			to.port = BigShort( ( short )( PORT_SERVER + j ) );
+			to.port = UBigShort( ( short )( PORT_SERVER + j ) );
```

Comment:
I think you would want unsigned/uint16_t here too

Reply:
all changes made